test/common/devCompileOptions.cmake

```
@@ -157,7 +157,12 @@ ELSE()
         IF(alpaka_ACC_GPU_HIP_ENABLE)
             LIST(APPEND alpaka_DEV_COMPILE_OPTIONS "-Wno-unused-command-line-argument")
         ENDIF()
-    # ICC
+        IF(CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM" OR
+            (CMAKE_VERSION VERSION_LESS 20 AND CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10))
```

Shouldn't this line look more like this:

```suggestion
            (CMAKE_VERSION VERSION_LESS 3.20 AND CMAKE_CXX_COMPILER_ID STREQUAL "Clang" AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 10))
```

---

Yes, it should :)